### PR TITLE
Add accuracy property to results

### PR DESCRIPTION
**Pull request by dianashk**
Each record in results should have an `accuracy` property. Accuracy can be any of the following:
- `point`
- `interpolated`
- `centroid`

In this release `interpolated` is not handled as we currently don't support that.

### CODE REVIEW

File: middleware/accuracy.js [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+ * Accuracy level should be set for each item in the results.
+ * The level can be any of the following:
+ *  - point
+ *  - interpolated
+ *  - centroid
+ */
+
+var check = require('check-types');
+
+var accuracyLevel_point = 'point';
```
Comment (missinglink) [inline middleware/accuracy.js]: variable can't decide if he's a camelCase of a lowercase with underscores :)

**Comment by trescube** [comment]
:shipit:

**Comment by dianashk** [comment]
I rebased this against my `confidence-score-fix` branch so you'll now see all those changes here, but this way there shouldn't be conflicts when we merge this after the confidence score change is merged.

**Comment by missinglink** [comment]
:shipit:

**Pull request merged**